docs/tutorial-script-single.rst

```diff
@@ -0,0 +1,280 @@
+    ha-demo-app                      Microsoft.Compute/virtualMachines                      eastus
+    ha-demo-b                        Microsoft.Compute/virtualMachines                      eastus
+    ha-demo-monitor                  Microsoft.Compute/virtualMachines                      eastus
+    ha-demo-appVMNic                 Microsoft.Network/networkInterfaces                    eastus
+    ha-demo-aVMNic                   Microsoft.Network/networkInterfaces                    eastus
+    ha-demo-bVMNic                   Microsoft.Network/networkInterfaces                    eastus
+    ha-demo-monitorVMNic             Microsoft.Network/networkInterfaces                    eastus
+    ha-demo-nsg                      Microsoft.Network/networkSecurityGroups                eastus
+    ha-demo-a-ip                     Microsoft.Network/publicIPAddresses                    eastus
+    ha-demo-app-ip                   Microsoft.Network/publicIPAddresses                    eastus
+    ha-demo-b-ip                     Microsoft.Network/publicIPAddresses                    eastus
+    ha-demo-monitor-ip               Microsoft.Network/publicIPAddresses                    eastus
+    ha-demo-net                      Microsoft.Network/virtualNetworks                      eastus
+
+.. _script_single_automated:
+
+Using the pg_autoctl Azure integration commands
+-----------------------------------------------
+
+The following command creates a new Azure resource group and then deploy
+some network settings (vnet, nsg, nsg rules, subnet), and then creates the
+target Virtual Machines and provision them:
+
+.. code-block:: bash
+
+   PG_AUTOCTL_DEBUG=1 pg_autoctl do azure create region \
+      --prefix ha-demo \
+      --region paris \
+      --location francecentral \
+      --monitor \
+      --nodes 2
+
+Adding to the command line shown above the option ``--script`` (not used in
+the previous example) allows a dry-run of the command that outputs a script
+that you can re-use later, or audit for details.
+
+In this command we are using the prefix ``ha-demo`` for our environment, and
+are naming our target region ``paris`` for our own reference. We chose to
+name it ``paris`` because our target Azure location in this example is
+``francecentral``. You could as well name your region ``nyc`` and choose to
+deploy your nodes in ``eastus`` for a much better latency if you happen to
+be connected from some place in the North American continent:
+
+.. code-block:: bash
+
+   PG_AUTOCTL_DEBUG=1 pg_autoctl do azure create region \
+      --prefix ha-demo \
+      --region nyc \
+      --location eastus \
+      --monitor \
+      --nodes 2
+
+Finaly, the option ``--from-source`` allows to provision the VM nodes from
+your current local pg_auto_failover source tree, using ``rsync`` to upload
+your local sources to the VMs:
+
+.. code-block:: bash
+
+   PG_AUTOCTL_DEBUG=1 pg_autoctl do azure create region \
+      --prefix ha-demo-dim \
```

Comment:
In the previous section where we ran commands by hand, we used `ha-demo`.

Why does this one change to `ha-demo-dim`?